site2/website/sidebars.json

```diff
@@ -120,6 +120,12 @@
       "cookbooks-message-queue",
       "cookbooks-bookkeepermetadata"
     ],
+    "Tiered Storage": [
+      "tiered-storage-overview",
+      "tiered-principle-of-offloaders",
+      "tiered-how-to-manage-offloaders",
+      "tiered-how-to-use-offloaders"
```

Comment:
A tiny suggestion for `id` in `sidebars.json`: give it a short name for easier maintenance.